src/v2/components/settings/PathSettings.tsx

```
@@ -0,0 +1,203 @@
+          {/* Photo Directory Section */}
+          <div className="space-y-2">
+            <label className="block text-sm font-medium text-gray-700 dark:text-gray-300">
+              {t('settings.paths.photoDirectory')}
+            </label>
+            <div className="flex gap-2">
+              <input
+                type="text"
+                value={photoDir?.value || ''}
+                readOnly
+                placeholder="/path/to/photos"
+                className="flex-1 block w-full rounded-md border border-gray-300 dark:border-gray-600 bg-white dark:bg-gray-800 px-3 py-2 text-sm text-gray-900 dark:text-white placeholder-gray-400 dark:placeholder-gray-500 focus:border-indigo-500 focus:ring-1 focus:ring-indigo-500"
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)